js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -366,11 +366,15 @@ const createIndicesHelper =
       const uniformPrefix = useUniform || uniformOnly ? 'uniforms.' : '';
       const shape = `${uniformPrefix}${name}_shape`;
       const strides = `${uniformPrefix}${name}_strides`;
+      const vecIndex = ['.x', '.y', '.z', '.w', '.a', '.b'];
+      for (let i = 0; i < rank - vecIndex.length; i++) {
+        vecIndex.push(`.${String.fromCharCode(93 + vecIndex.length + i)}`);
```

Comment:
When rank is big enough, will it produce repeating member, like `['x', 'y', ... ... '.x', '.y'...]`?

Reply:
Add error when > 26.

Reply:
Add error when > 26.